src/web_interface/templates/user_management/edit_user.html

```diff
@@ -0,0 +1,161 @@
+
+
+{% block styles %}
+<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bootstrap3-editable/css/bootstrap-editable.css') }}">
+<script type="text/javascript" src="{{ url_for('static', filename='bootstrap3-editable/js/bootstrap-editable.min.js') }}"></script>
+<style>
+    .editable-container.editable-inline,
+    .editable-container.editable-inline .control-group.form-group,
+    .editable-container.editable-inline .control-group.form-group .editable-input,
+    .editable-container.editable-inline .control-group.form-group .editable-input textarea,
+    .editable-container.editable-inline .control-group.form-group .editable-input select,
+    .editable-container.editable-inline .control-group.form-group .editable-input input:not([type=radio]):not([type=checkbox]):not([type=submit])
+    {
+        width: 100%!important;
+    }
+</style>
+
+<script>
+    $.fn.editable.defaults.mode = 'inline';
+    $(document).ready(function() {
+        $('#roles').editable({
+            value: {{ role_indexes }},
+            source: [
+                {% for r in available_roles %}
+                    {value: {{ available_roles.index(r) }}, text: '{{ r }}'},
+                {% endfor %}
+            ]
+        });
+    });
+</script>
+{% endblock %}
+
+
+{% block body %}
+
+<div class="row" style="margin-bottom: 20px">
+    <div class="col-sm-12 col-md-8 col-md-offset-2">
+        <h2>
+            Manage User '{{ user.email }}'
+        </h2>
+    </div>
+</div>
+
+<div class="row">
+    <div class="col-sm-12 col-md-8 col-md-offset-2">
+        <table class="table table-bordered">
+
+            <tr>
+                <td class="active result" width="120px" style="text-align: right;">
+                    Username:
+                </td>
+                <td class="active result">
+                    {{ user.email }}
+                </td>
+            </tr>
+
+            <tr>
+                <td class="active result" style="text-align: right;">
+                    ID:
+                </td>
+                <td class="active result">
+                    {{ user.id }}
+                </td>
+            </tr>
+
+            <tr>
+                <td class="active result" style="text-align: right;">
+                    API Key:
+                </td>
+                <td class="active result">
+                    {{ user.api_key }}
+                </td>
+            </tr>
+
+            <tr>
+                <td class="active result" style="text-align: right;">
+                    Active:
+                </td>
+                <td class="active result">
+                    {{ user.active }}
+                </td>
+            </tr>
+
+            <!-- user roles -->
+            <tr>
+                <td class="active result" style="text-align: right; vertical-align: middle;">
+                    Roles:
+                </td>
+                <td class="active result" style="padding-right: 90px">
```

Comment:
Is it possible to add tool tips indicating the privileges of a role? That would be nice.